images/index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Beautiful Free Nova template</title>
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap">
+</head>
+<body>
+  <div class="green">
+    <header class="top">
+        <img src="logo.svg" class="logo">
+        <nav>
+            <ul class="menu">
+                <li><a href="#">The Team</a></li>
+                <li>
+                    <a href="#">Pricing</a>
+                </li>
+                <li><a href="#">Features</a></li>
+                <li><a class="button" href="#">DONWLOAD FREE!</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="article">
+        <div class="col">
+            <h1 font-size:36px, font-weight: 600>Beautiful Free Nova template</h1>
+            <p class="header-text"> A top notch premium quality template for your next web project.</p>
+            <h4> Download FREE!</h4>
+        </div>
+        <div class="col">
+            <img src="screen.png" class="ipad-screen">
+        </div>
+    </section>
+
+  </div> <!-- ten zielony -->
+ <section class="black-section">
+    <h1> FEATURES </h1>
+    <div class="container2">
+        <div class="feature">
```

Comment:
Zwróć uwagę, że mamy tutaj "listę cech" dlatego zamiast `div > div` zrobiłbym `ul > li` - będzie to czytelniejsze